docs/md/docs/2020-11-02-turn-the-senior-around.md

```diff
@@ -0,0 +1,68 @@
+The senior engineer's role is first and foremost to constantly give feedback to the non senior engineer who leads the feature - help them learn by providing them a great ground for them to grow.<br>
+The team lead's job is double:
+- Communicate with the non seniors as feature leads, help if needed and make sure everything is on track
+- Communicate with the senior engineers and give them all the help needed to __*make sure they have all they need to grow the non senior members*__. The team lead constantly gives feedback to the senior engineers on how they improve others.
+
+## Advantages Of Turning The Senior Around
+### Dynamic Team Hierarchy
+The non seniors feel that the limit to their performance is only themselves and not the senior who leads them (in the scenario before). The fact that the non seniors lead the feature provides them the needed experience and confidence to later become seniors themselves.
+
+### Maximal Growth
+* From the senior engineer's perspective: they are focused on making others better. They constantly get feedback from the team lead on their ability to do so while of course being active part of the feature implementation.<br>
+The senior engineers still use and maintain their strong muscles (design, coding etc) by mentoring the non seniors - pairing, feedback etc.
+* From the non senior engineer's perspective: they grow by trying their best to lead a feature while getting constant feedback from the senior team members. They are exposed to all of the "senior qualities" in first hand and improve on a daily basis.
+
+### Long Term Team Building
+When you zoom out and see what happens to a team that uses this method for a long time you see strong teams where all the team members maximize their growth in their respective areas of learning.<br>
+You see a stronger team where seniors constantly grow the non seniors, leading to a robust team.<br>
+You see a team that promotes more people to seniors.<br>
+You see a team where senior engineers can break through the senior limits and get promoted themselves.<br>
+You see a team with satisfied team members :)<br>
+
+## The Risk
+Having said all of that there is a short term risk here - you put all your faith and trust on the non senior members to lead critical features and rely mostly on the seniors to help them.<br>
+This requires a huge trust in the team and an ability of the team lead to maximize the empowerment of the team.<br>
+I personally really enjoy this step, when you trust your team this way the team feels it - this ability to rely on the team while having long term team building vision is exactly where we grow as engineering managers, but this might be a topic to another blog post :) <br>
+
+# Summary
+Turning the senior around means to flip the team structure - let the non seniors lead features and make sure the senior members grow them by constant feedback.<br>
```

Comment:
This sentence sounds like it could use some rephrasing.
`Turning the senior engineer around also means flipping team structure around by letting non-senior engineers lead features while senior engineers help them grow by constantly providing feedback.`